client/.eslintrc.json

```diff
@@ -1,3 +1,3 @@
 {
-  "extends": ["next/babel", "next/core-web-vitals"]
```

Comment:
The issue of removing the `next/babel` dependency is that it pops up as a line error in VSCode (just the editor, not while executing the code). I'm okay with having/removing it, however.

Reply:
it's not being removed, just being rewritten this other way to avoid one error I was getting while doing ci setup.